app/models/channel.rb

```diff
@@ -41,6 +41,10 @@ def self.find_by_permalink!(string)
     self.by_permalink(string).first!
   end
 
+  def updatable_recurring?
```

Comment:
I think this reflects more as `cancelable_recurring?` than the `updatable`, because you can update the recurring to true and pass this validation...what do you think?